srcpkgs/python3-pytest-sugar/template

```diff
@@ -12,7 +12,7 @@ license="BSD-3-Clause"
 homepage="http://pivotfinland.com/pytest-sugar"
```

Comment:
this homepage is empty, can we switch to github or something else?

Reply:
done